apps/fluent-tester/package.json

```diff
@@ -92,45 +93,35 @@
     "@rnx-kit/metro-config": "^1.2.26",
     "@types/jasmine": "3.5.10",
     "@types/react": "^17.0.2",
-    "@types/react-native": "^0.66.0",
+    "@types/react-native": "^0.68.0",
     "@wdio/appium-service": "7.23.0",
     "@wdio/cli": "7.23.0",
     "@wdio/jasmine-framework": "7.23.0",
     "@wdio/local-runner": "7.23.0",
     "@wdio/spec-reporter": "7.23.0",
     "appium": "2.0.0-beta.41",
     "appium-mac2-driver": "1.4.0",
-    "appium-xcuitest-driver": "4.11.1",
     "appium-windows-driver": "2.0.7",
+    "appium-xcuitest-driver": "4.11.1",
     "flow-bin": "^0.113.0",
-    "metro-config": "^0.66.2",
-    "metro-react-native-babel-preset": "^0.66.2",
-    "react": "17.0.2",
-    "react-native": "^0.66.0",
-    "react-native-macos": "^0.66.0",
-    "react-native-svg": "^12.3.0",
+    "metro-config": "^0.67.0",
+    "metro-react-native-babel-preset": "^0.67.0",
     "react-native-svg-transformer": "^1.0.0",
-    "react-native-test-app": "^1.6.2",
-    "react-native-windows": "^0.66.0",
+    "react-native-test-app": "^1.3.5",
     "react-test-renderer": "17.0.2",
     "ts-node": "^8.10.1",
     "tsconfig-paths": "^3.9.0",
     "typescript": "4.5.4",
     "webdriverio": "7.23.0"
   },
   "peerDependencies": {
-    "@office-iss/react-native-win32": "^0.66.6",
-    "react": "17.0.2",
-    "react-native": "^0.66.0",
-    "react-native-macos": "^0.66.0",
-    "react-native-svg": "^12.3.0",
-    "react-native-windows": "^0.66.0"
```

Comment:
Just double checking - does it make sense to remove all these dependencies from `peerDependencies` in the fluent-tester package?

Reply:
If they're all being moved to dependencies then yeah, that should be fine.